### PR TITLE
[Update] Installation and Update content to include MacPorts method

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

This PR has instructions to install and update timescaledb and postgresql using MacPorts

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/757

### CODE REVIEW

File: install/installation-macos.md [hunk 1]
```diff
@@ -38,81 +44,33 @@ instead.
...
-you might have used to connect to PostgreSQL before, but if you haven't
-installed it yet, check out our [installing psql][install-psql] section.
-
 <procedure>
 
-### Setting up the TimescaleDB extension
-1.  On your local system, at the command prompt, connect to the PostgreSQL
-    instance as the `postgres` superuser:
+#### Using MacPorts
+1.  Install MacPorts, by downloading the macOS package and double-click the downloaded package installer.
```
Comment (charislam) [inline install/installation-macos.md:50]:
```suggestion
1.  Install MacPorts by downloading and running the package installer.
```

File: install/installation-macos.md [hunk 2]
```diff
@@ -38,81 +44,33 @@ instead.
...
-1.  Connect to the database you created:
-    ```sql
-    \c example
-    ```
-1.  Add the TimescaleDB extension:
-    ```sql
-    CREATE EXTENSION IF NOT EXISTS timescaledb;
-    ```
-1.  You can now connect to your database using this command:
+1.  View the files that were installed by timescaledb:
```
Comment (charislam) [inline install/installation-macos.md:58]:
```suggestion
1.  View the files that were installed by TimescaleDB:
```

File: install/installation-macos.md [hunk 3]
```diff
@@ -38,81 +44,33 @@ instead.
...
     ```bash
-    psql -U postgres -h localhost -d example
-    ```
+    port contents timescaledb
+    ``` 
+     
+<highlight type="warning">
+ MacPorts does not install the `timescaledb-tools` to run the `timescaledb-tune` script. For more information about installing and using the tool, see [timescaledb-tune][tune] section.
```
Comment (charislam) [inline install/installation-macos.md:64]:
```suggestion
 MacPorts does not install the `timescaledb-tools` to run the `timescaledb-tune` script. 
 For more information about installing and using the tool, see the [timescaledb-tune][tune] section.
```

File: timescaledb/how-to-guides/connecting/psql.md [hunk 4]
```diff
@@ -28,14 +28,13 @@ wmic
 
 </terminal>
 
-## Install psql on Apple macOS
+## Install psql on macOS
 The `psql` tool is installed by default on macOS systems when you install
-PostgreSQL, and this is the most effective way to install the tool. These
-instructions use the Homebrew package management tool.
+PostgreSQL, and this is the most effective way to install the tool. On macOS you can use Homebrew or MacPorts to install PostgreSQL package or just the `psql` tool.
```
Comment (charislam) [inline timescaledb/how-to-guides/connecting/psql.md:33]:
```suggestion
PostgreSQL, and this is the most effective way to install the tool. On macOS 
you can use Homebrew or MacPorts to install the PostgreSQL package or just the `psql` tool.
```

File: timescaledb/how-to-guides/connecting/psql.md [hunk 5]
```diff
@@ -83,13 +102,29 @@ If you do not want to install the entire PostgreSQL package, you can install the
...
+</procedure>
+
 ## Install psql on Debian and Ubuntu
 You can use the `apt` package manager on Debian and Ubuntu systems to install
 the `psql` tool.
 
 <procedure>
 
-### Installing psql on Debian and Ubuntu
+### Installing psql using package manager
```
Comment (charislam) [inline timescaledb/how-to-guides/connecting/psql.md:127]:
```suggestion
### Installing psql using the apt package manager
```

**Review by charislam — APPROVED** [review]
There's also a mention of "Homebrew installer" on line 3 that probably needs MacPorts added :)

File: timescaledb/how-to-guides/connecting/psql.md [hunk 6]
```diff
@@ -54,11 +53,31 @@ instructions use the Homebrew package management tool.
...
+    For example, to install version *14* replace `postgresql<xx>` with `postgresql14`.
+1.  View the files that were installed:
+    ```bash
+    port contents postgresql<xx>
+    ``` 
+
+</procedure>
+
+### Installing psql on macOS
+If you do not want to install the entire PostgreSQL package, you can install the `psql` tool on its own. `libpqxx` is the official C++ client API for PostgreSQL, the enterprise-strength open-source database software.
```
Comment (charislam) [inline timescaledb/how-to-guides/connecting/psql.md:75]:
```suggestion
If you do not want to install the entire PostgreSQL package, you can install the `psql` tool on its own. `libpqxx` is the official C++ client API for PostgreSQL.
```

**Review by charislam — APPROVED** [review]

File: install/installation-macos.md [hunk 7]
```diff
@@ -38,81 +41,37 @@ instead.
...
     ```bash
-    psql -U postgres -h localhost -d example
+    port contents timescaledb
     ```
 
+<highlight type="important">
+By default, MacPorts does not install the `timescaledb-tools` package, so the
+`timescaledb-tune` script is not included. For more information about installing
+and using the tool, see the
+[timescaledb-tune](/timescaledb/:currentVersion:/how-to-guides/configuration/timescaledb-tune/#timescaledb-tuning-tool) section.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply install/installation-macos.md:66]: Oh, good spot! I got halfway there 😉

**Comment by Loquacity** [comment]
@Rajakavitha1 How close are we to be able to merge this, do you think?

**Review by Loquacity — APPROVED** [review]

File: install/installation-macos.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Install self-hosted TimescaleDB on macOS systems
 You can host TimescaleDB yourself on your Apple macOS system.
-These instructions use a Homebrew installer on these versions:
+These instructions use a Homebrew or Macports installer on these versions:
```
Comment (svenklemm) [inline install/installation-macos.md:3]:
```suggestion
These instructions use a Homebrew or MacPorts installer on these versions:
```

**Pull request merged**